packages/story-editor/src/components/richText/karma/singleSelection.karma.js

```diff
@@ -169,7 +164,6 @@ describe('Styling single text field', () => {
       await data.fixture.events.click(await fontWeight.option('Bold'));
       await data.fixture.events.sleep(300);
       await data.fixture.events.click(fontColor.button);
-      waitFor(() => fontColor.picker);
```

Comment:
This was for waiting for the Color Picker to open, curious if anything changed that it's not needed anymore, or if it wasn't needed before either?

Reply:
The `waitFor` here wasn't actually doing anything. Since `font.picker` doesn't throw an error, the `waitFor` never repolls. `waitFor` doesn't care if `font.picker` is null or not. If we think we do need to wait, we can add the same format as some of the other changes here. ie 
```
await waitFor(() => {
  if (!something) {
   throw new Error('this will make it so that the waitFor polls until no error is thrown')
  }
  expect(font.picker).toBeTruthy();
}
```

Reply:
> The waitFor here wasn't actually doing anything.

Ah, okay, I thought that `waitFor` would first do the initial interval wait and then run the callback, but it looks like it's running the callback first.

Reply:
We can use `waitFor` without manually throwing errors _if_ the `expect` (or anything) inside throws an error for us. 
https://testing-library.com/docs/dom-testing-library/api-async/